webapp/nuxt.config.js

```diff
@@ -38,9 +47,17 @@ export default {
 
   // Modules: https://go.nuxtjs.dev/config-modules
   modules: [
+    '@nuxtjs/apollo',
   ],
 
+  apollo: {
+    clientConfigs: {
+      default: {
+        httpEndpoint: 'http://localhost:4000',
+      }
+    }
+  },
+
   // Build Configuration: https://go.nuxtjs.dev/config-build
-  build: {
-  }
+  build: {},
```

Comment:
🔴 Maybe I am overlooking something, but I guess PWA is still missing.
Remember to catch your ⭐ 

> For Lighthouse reporting that your production website is installable as PWA (except HTTPS).